Examples/LambdaFunctions/Sources/LambdaAPI/ProductLambdaHandler.swift

```diff
@@ -0,0 +1,222 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+import AWSLambdaRuntime
+import NIO
+import Logging
+import AWSLambdaEvents
+
+struct ProductLambdaHandler: EventLoopLambdaHandler {
+    
+    typealias In = APIGateway.V2.Request
+    typealias Out = APIGateway.V2.Response
+    
+    let service: ProductService
+    let operation: Operation
+    
+    func handle(context: Lambda.Context, event: APIGateway.V2.Request) -> EventLoopFuture<APIGateway.V2.Response> {
+    
+        switch operation {
+        case .create:
+            logger.info("create")
```

Comment:
more meaningful logging? also debug instead of info?

Reply:
e123d60